source/browseMode.py

```diff
@@ -628,6 +628,25 @@ def addQuickNav(
 		if key is not None:
 			cls.__gestures["kb:shift+%s" % key] = scriptName
 
+	@classmethod
+	def _addQuickNavHeading(
+		cls,
+		levelRange: range,
+	):
+		for i in levelRange:
+			cls.addQuickNav(
+				f"heading{i}",
+				key=f"{i}",
+				# Translators: Input help message for a Word quick navigation command in browse mode.
+				nextDoc=_("Moves to the next heading at level {i}").format(i=i),
+				# Translators: Message presented when the browse mode element is not found.
+				nextError=_("No next heading at level {i}").format(i=i),
+				# Translators: Input help message for a quick navigation command in browse mode.
```

Comment:
Document the substitution
```suggestion
				# Translators: Input help message for a quick navigation command in browse mode.
				# {i} will be replaced with the level number.
```